pages/workflow/index.php

```diff
@@ -24,7 +24,5 @@
     case 'externalReview':
     case 'editorial':
     case 'production':
```

Comment:
I am only familiar with access&index. Not sure when the others are used.

Reply:
I figured we should leave those in place because they could conceivably be bookmarked or included in emails.